python/lsst/ip/isr/fringe.py

```diff
@@ -115,7 +115,7 @@ def loadFringes(self, fringeExp, expId=None, assembler=None):
         if expId is None:
             seed = self.config.stats.rngSeedOffset
         else:
-            print(f"{self.config.stats.rngSeedOffset} {expId}")
+            self.log.debug("Seeding with offset %d and expId %d.", self.config.stats.rngSeedOffset, expId)
```

Comment:
One minor suggestion to help parsing the log output.

```suggestion
            self.log.debug("Seeding with offset %d and exposureId %d.", self.config.stats.rngSeedOffset, expId)
```

Reply:
Oh, yes, definitely!

Reply:
Ugh... I've just bounced back and forth on this twice. I think `ccdExposureId` is correct, since that's what it comes from, and ISR runs per-detector, and we should be explicit in the logs.